substrate/frame/staking/reward-curve/src/lib.rs

```diff
@@ -81,11 +81,12 @@ pub fn build(input: TokenStream) -> TokenStream {
 
 	let imports = match crate_name("sp-runtime") {
 		Ok(FoundCrate::Itself) => quote!(
-			extern crate sp_runtime as _sp_runtime;
+			#[doc(hidden)]
+			pub use crate as _sp_runtime;
```

Comment:
This doesn't look correct; originally the external crate is `sp_runtime`, so the code really should look like:

```suggestion
			pub use sp_runtime as _sp_runtime;
```

Reply:
yes thanks!